airbyte-integrations/connectors/source-okta/acceptance-test-config.yml

```diff
@@ -3,7 +3,7 @@ test_strictness_level: high
 acceptance_tests:
   spec:
     tests:
-      - spec_path: "source_okta/spec.json"
+      - spec_path: "source_okta/spec.yaml"
```

Comment:
It doesn't, because when we migrate it to lowcode, we change the json file to yaml, and spec gets inserted in the manifest.yaml file. 
Should I change it to manifest.yaml? Or should I duplicate the spec in a new file spec.yaml?

Reply:
Should I just remove the spec_path field then?

Reply:
Removed the block as the file didn't exist.

Reply:
This is resolved. I merged the latest changes from master and it now shows the full expected and current schema.